src/Microsoft.AspNet.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -89,7 +88,8 @@ public OAuthHandler(HttpClient backchannel)
                 var value = query["error"];
                 if (!StringValues.IsNullOrEmpty(value))
                 {
-                    Logger.LogVerbose("Remote server returned an error: " + Request.QueryString);
+                    Logger.LogWarning("Remote server returned an error: " + value);
```

Comment:
Yeah, that would be great! But I guess it's too late to include these changes in beta8?